### PR TITLE
Adding cors support and default

### PR DESCRIPTION
**Pull request by nicola**
- [x] cors on apps
- [x] tests
- [x] make it identical to gold
- [x] OPTIONS support

### CODE REVIEW

**Comment by deiu** [comment]
Where is the test? :)

**Comment by nicola** [comment]
this is the right question, coming

**Comment by nicola** [comment]
I just realized looking at Gold, that this library is not enough, since it only sets `Allow-Control-Origin: *`

File: index.js [hunk 1]
```diff
@@ -41,6 +42,19 @@ function ldnode (argv) {
     // Setting options as local variable
     app.locals.ldp = opts;
 
+    // Setting CORS
+    app.use(cors({
+        methods: [
+            'OPTIONS', 'HEAD', 'GET',
+            'PATCH', 'POST', 'PUT', 'MKCOL', 'DELETE',
+            'COPY', 'MOVE', 'LOCK', 'UNLOCK'
```
Comment (deiu) [inline index.js]: Are all those methods supported by ldnode? MKCOL, COPY, MOVE, LOCK, UNLOCK are methods specific to webdav.

Comment (nicola) [inline index.js]: These methods are not supported yet. I just wanted to align them with gold, so I will just remove them. I am implementing options at the moment.

Comment (nicola) [inline index.js]: Actually cors() implements a trivial 'OPTIONS' handler by default - but we have to linkeddata-ize it!

**Pull request merged**